src/GovUk.Education.ExploreEducationStatistics.Notifier/Functions/ApiSubscriptionFunctions.cs

```diff
@@ -45,7 +45,7 @@ public class ApiSubscriptionFunctions(
 
     [Function("VerifyApiSubscription")]
     public async Task<IActionResult> VerifyApiSubscription(
-        [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "public-api/{dataSetId:guid}/verify-subscription/{token}")]
+        [HttpTrigger(AuthorizationLevel.Anonymous, "patch", Route = "public-api/{dataSetId:guid}/verify-subscription/{token}")]
```

Comment:
Controversial one but my gut feeling is that this should be a `POST `since a `PUT` or a `PATCH` both expect a document in the body representing the resource or partial resource to be updated.